hm_11/hm_011.py

```diff
@@ -0,0 +1,61 @@
+# task_num_1
+import math
+import datetime
+
+
+def get_time_and_name(func):
+    def inner(*args, **kwargs):
+        call_time = datetime.datetime.now()
+        func(*args, **kwargs)
```

Comment:
You can assign the result of the func call to some variable and return it in the end to be able to decorate function that return some value